### ISSUE TITLE
Fix console output in test suite

### ISSUE THREAD

**Issue by murny** [ualbertalib/jupiter#2106]
Seems like  a recent change has started polluting our test runner (`rails test`). As shown here:

```
ERROR [line 20]      Lexer error: With input '<rs:ln href="/items/e2ec88e3-3266-4e95-8575-8b04fac2a679/view/c7732455-1cff-45c8-b22b-d046a7fd7181/im': Invalid token "<rs:ln": {:production=>:path, :token=>"<rs:ln"}: 
ERROR [line 22]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 23]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 24]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 25]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 26]      Expected objectList: {:production=>:predicateObjectList, :token=>"."}: 
ERROR [line 20]      Lexer error: With input '<rs:ln href="/items/e2ec88e3-3266-4e95-8575-8b04fac2a679/view/c7732455-1cff-45c8-b22b-d046a7fd7181/im': Invalid token "<rs:ln": {:production=>:path, :token=>"<rs:ln"}: 
ERROR [line 22]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 23]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 24]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 25]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 26]      Expected objectList: {:production=>:predicateObjectList, :token=>"."}: 
ERROR [line 20]      Lexer error: With input '<rs:ln href="/items/8e18f37c-dc60-41bb-9459-990586176730/view/f803c425-7424-4f4a-bda0-1c0908702ea1/im': Invalid token "<rs:ln": {:production=>:path, :token=>"<rs:ln"}: 
ERROR [line 22]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 23]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 24]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 25]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 26]      Expected objectList: {:production=>:predicateObjectList, :token=>"."}: 
ERROR [line 20]      Lexer error: With input '<rs:ln href="/items/8e18f37c-dc60-41bb-9459-990586176730/view/f803c425-7424-4f4a-bda0-1c0908702ea1/im': Invalid token "<rs:ln": {:production=>:path, :token=>"<rs:ln"}: 
ERROR [line 22]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 23]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 24]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 25]      Expected objectList: {:production=>:predicateObjectList, :token=>";"}: 
ERROR [line 26]      Expected objectList: {:production=>:predicateObjectList, :token=>"."}: 
```

**Comment by pgwillia** [comment]
Might have been this https://github.com/ualbertalib/jupiter/pull/2070

**Comment by murny** [comment]
Ah good find 👍 . Maybe their's something we can do to fix it? Otherwise maybe we can stub out their output/logging and just silence this 🤔 . Needs some more investigation

**Comment by pgwillia** [comment]
Agreed! Needs some more investigation.  @lagoan might know more about this error, what it indicates, and if it can be ignored?

**Comment by lagoan** [comment]
I will take a look at it.

**Comment by mbarnett** [comment]
Just ran into this today, and bisected it back to the PR referencing this issue – I'm thinking these errors aren't benign and that rdf-n3 3.1.2 broke more than just that method alias (side note: it's really concerning that these RDF gems continue to be so bad about sticking to SemVer and we may need to start excluding them from dependabot if they can't get their act together, because this ought to have been a 4.0.0 for that method removal)

Going to open a PR to pin us back to rdf-n3 3.1.1 until we have a better sense of whether or not this has actually broken things in a way that are just not causing tests to fail or not.

**Comment by murny** [comment]
Believe this is fixed now, thanks to Matts PR #2199 that is now merged in. Closing issue 👍

**Comment by lagoan** [comment]
Reopening after creating a PR that fixes the error using the newer version of gem rdf-n3

**Comment by lagoan** [comment]
Fixed on #2242
